include/oneapi/mkl/dft/detail/commit_impl.hpp

```diff
@@ -26,25 +26,40 @@
 #include <CL/sycl.hpp>
 #endif
 
+#include "oneapi/mkl/detail/backends.hpp"
+
 namespace oneapi {
 namespace mkl {
 namespace dft {
 namespace detail {
 
 class commit_impl {
 public:
-    commit_impl(sycl::queue queue) : queue_(queue), status(false) {}
+    commit_impl(sycl::queue queue, mkl::backend backend)
+            : queue_(queue),
+              backend_(backend),
+              status(false) {}
 
-    commit_impl(const commit_impl& other) : queue_(other.queue_), status(other.status) {}
+    commit_impl(const commit_impl& other)
+            : queue_(other.queue_),
+              backend_(other.backend_),
+              status(other.status) {}
 
     virtual ~commit_impl() {}
```

Comment:
I think I prefer to use "= default". Makes it more clear. Just a preference though. Feel free to keep it as is. Same comment would apply to the derived classes

Reply:
Done in [d892e17](https://github.com/hjabird/oneMKL/pull/4/commits/d892e177b3bd2f559fbde3cc4b2684008abfac2d).